src/jdk.naming.rmi/share/classes/module-info.java

```diff
@@ -58,8 +58,9 @@
  *          implementation.
  *     </li>
  * </ul>
- * <p> Factory class names associated with a {@linkplain javax.naming.Reference#getFactoryClassLocation() factory
- * class location} are not supported out of the box. Unless an {@link javax.naming.spi.ObjectFactoryBuilder
+ * <p> Downloading a factory class from a {@linkplain javax.naming.Reference#getFactoryClassLocation()
+ * location} specified in the reference can be supported by a custom implementation of {@link
+ * javax.naming.spi.ObjectFactoryBuilder}. Unless an {@link javax.naming.spi.ObjectFactoryBuilder
```

Comment:
```suggestion
 * javax.naming.spi.ObjectFactoryBuilder}. If a location is specified, then 
 * unless an {@link javax.naming.spi.ObjectFactoryBuilder
```

Reply:
Thank you changed in [991994d](https://github.com/openjdk/jdk/pull/22154/commits/991994da64bcc5c2f8bfb5cf8704e926948153f0)